modules/ROOT/pages/lb-create-arm.adoc

```diff
@@ -53,8 +53,17 @@ Accepts the inbound request on the default SSL endpoint using the HTTP protocol.
 * *Redirect*
 +
 Redirects the request to the same URL using the HTTPS protocol.
+. Specify options:
++
+* *Disable Static IPs* specifies to use dynamic IPs, which do not persist when the DLB restarts.
+* *Keep URL encoding* specifies to encode the space (`%20`) and hash (`%23`) special characters.
++ 
+[IMPORTANT]
+The DLB only encodes the `%20` and `%23` characters. If a requesting URL includes other special characters, they are not encoded and result in errors, such as `Illegal character in path`. See https://help.mulesoft.com/s/article/Some-Special-Characters-Do-Not-Work-in-URLs-to-DLB[DLB and Special Characters].
+* *Support TLS 1.0* supports Transport Layer Security (TLS) v1 encryption in addition to TLS v1.1 and TLS v1.2.
+* *Upstream TLS 1.2* enables TLS v1.2 encryption on upstream traffic.
```

Comment:
You may ant to specify that this option will enable tls v1.2 to the mule workers behind the DLB, it may not be clear to the user.

Reply:
Thanks @henrymollman - that wasn't clear to me! Updated.